controls/listview/features/listview-features-selection.md

```diff
@@ -13,78 +13,42 @@ The **RadListView** component exposes selection feature. It allows users to sele
 	- Tap
 	- Hold
 - **SelectionMode** (*SelectionMode*): Gets or sets the mode of the selection.
-	- None
-	- Single
-	- Multiple
+	- None - This mode doesn't allow users to select an item. To enable it the **SelectionMode** property should be set to **None**.
+	- Single - This is the default selection mode. It allows users to select only one item.
+	- Multiple - This mode allows users to select more than one item. To enable it the **SelectionMode** property should be set to **SelectionMode.Multiple**.
+	
 - **SelectedItems** (*ObservableCollection&lt;object&gt;*): Gets or sets the currently selected items.
 - **SelectedItem** (*object*): Gets the last selected item.
 - **SelectionChanged**: An event that is triggered whenever the SelectedItems collection is changed.
 
-## Single Selection
 
-This is the default selection mode. It allows users to select only one item. This is how it looks.
+## Example
 
-Figure 1: Single Selection
+Here is the definition of the ListView control:
 
-![SingleSelection](images/listview-features-selection-single.png "Single Selection")
+<snippet id='listview-features-selection-xaml'>
+
+Lets create a simple data for the ListView component and a method called InitializePickers in which we can add the modes and gestures we want to use:
+<snippet id='listview-features-selection-csharp'>
 
-## Example
 
-#### XAML
+After this lets create **OnSelectionGestureChanged** and **OnSelectionModeChanged** methods that will be invoked when the selection changes:
```

Comment:
Actually, these method are related to the Picker functionality and they are used to change the SelectionGesture/SelectionMode of the RadListView. So basically, when changing the selection of the Pickers, we are controlling the behavior of RadListView's selection. Please reconsider this sentence.